srcpkgs/avidemux/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'avidemux'
 pkgname=avidemux
 version=2.7.1
-revision=4
+revision=5
 wrksrc="${pkgname}_${version}"
+# Can't be compiled for aarch64, arm* or mips*
```

Comment:
doesn't fly, will leave those arches in staging.

Reply:
What is staging in Void?

Reply:
Packages are added to a staging repository, only if all shared libraries in the staging repository are consistent and there are no missing dependencies the main repository is updated.

Excluding `avidemux` would result in the old package depending on the old `x265` and other packages depending on a newer `x265` version resulting in a state that is not consistent, because there can be only one version of a package in the repository.

Reply:
But looking at the diff again, it had `only_for_archs` already. In this case there shouldn't be an old `avidemux` so this shouldn't be an issue.

Reply:
But avidemux isn't in the repos of these architectures, I just moved these lines to pass xlint.

Reply:
Ah ok

Reply:
Yes Right.

Again I'm not a fan of the new `xlint` variable order thing. In many cases, just produces larger diffs and confusion for many variables. https://github.com/chneukirchen/xtools/pull/115, `only_for_archs` is another variable where I think it shouldn't be necessary to force an order.